gapic/templates/%namespace/%name_%version/%sub/services/%service/transports/grpc.py.j2

```diff
@@ -48,8 +47,7 @@ class {{ service.name }}GrpcTransport({{ service.name }}Transport):
             credentials_file: str = None,
             scopes: Sequence[str] = None,
             channel: grpc.Channel = None,
-            api_mtls_endpoint: str = None,
-            client_cert_source: Callable[[], Tuple[bytes, bytes]] = None,
+            ssl_channel_credentials: grpc.ChannelCredentials = None,
```

Comment:
I added back the old kwargs and behavior, and a deprecation warning. So now there is no public interface breaking change.

Reply:
@software-dov I only added back the old kwargs and behavior etc to the regular template only. I didn't add those to ads template, they won't be using any mtls related stuff in at least couple of months from now since it is not available yet.

Reply:
Sounds good to me.